src/main/kotlin/com/semgrep/idea/telemetry/SentryProcessListener.kt

```diff
@@ -0,0 +1,36 @@
+package com.semgrep.idea.telemetry
+
+import com.intellij.execution.process.ProcessEvent
+import com.intellij.execution.process.ProcessListener
+import com.intellij.openapi.util.Key
+import io.sentry.Attachment
+import io.sentry.Hint
+import java.time.format.DateTimeFormatter
+
+class SentryProcessListener : ProcessListener {
+
+    private var processLog = mutableListOf<String>()
+    private var sentry = SentryWrapper.getInstance()
+
+    private fun attachmentOfLog(): Attachment {
+        val humanReadableTimestamp = DateTimeFormatter.ISO_INSTANT.format(java.time.Instant.now())
+        return Attachment(processLog.joinToString("\n").toByteArray(), "lsp-output-$humanReadableTimestamp.log")
```

Comment:
similar to vscode, can we use an actual file instead?

Reply:
yea good catch